src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const totalCost = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return totalCost - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return totalCost - SHORT_TERM_DISCOUNT;
```

Comment:
The logic here is correct in that it checks for the short-term discount only if the long-term discount is not applicable. However, ensure that the long-term discount is prioritized, which is already handled by the preceding condition. No changes needed here, but ensure the logic flow is clear.